charts/zipkin/templates/tests/test-connection.yaml

```diff
@@ -16,13 +16,31 @@ spec:
       # Make sure zipkin registered in Eureka at startup
       args: [ 'wget -q --spider http://{{ include "zipkin.fullname" . }}:8761/eureka/v2/apps/ZIPKIN' ]
 {{- end }}
+{{ if .Values.global.testDependencies }}
+    - name: get-frontend
+      image: "ghcr.io/openzipkin/alpine:3.19.1"
+      command: [ '/bin/sh', '-c' ]
+      args:  # Get the frontend service 3 times, after it is ready
+        - >
+           while ! wget -q --spider http://{{ include "zipkin.fullname" . }}:8081/health;do sleep 1; done &&
+           for i in 1 2 3;do wget -q --spider http://{{ include "zipkin.fullname" . }}:8081; done
+    - name: get-dependencies
```

Comment:
The follow-up is.. we'll need a conditional here to invoke the dependencies cron job synchronously, when elasticsearch.